CodeSniffer/Standards/Squiz/Sniffs/Commenting/FunctionCommentSniff.php

```diff
@@ -238,6 +246,22 @@ protected function processThrows(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $co
      */
     protected function processParams(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $commentStart)
     {
+        if ($this->_platform === null) {
```

Comment:
upon more careful review, $phpcsFile is not available in a constructor. This is why similar features like `tabWidth` do it like I have done here. If there is indeed a way, I'd be happy to use it.